presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -77,10 +77,10 @@ private static void populateDataToHiveTable(String destination, String source, S
     {
         String queryStatement = format("INSERT INTO TABLE %s PARTITION (%s) SELECT * FROM %s", destination, partition, source);
 
-        onHive().executeQuery("SET mapreduce.map.memory.mb=512");
-        onHive().executeQuery("SET mapreduce.map.java.opts=-Xmx384m");
-        onHive().executeQuery("SET mapreduce.reduce.memory.mb=512");
-        onHive().executeQuery("SET mapreduce.reduce.java.opts=-Xmx384m");
+        onHive().executeQuery("SET mapreduce.map.memory.mb=256");
+        onHive().executeQuery("SET mapreduce.map.java.opts=-Xmx196m");
+        onHive().executeQuery("SET mapreduce.reduce.memory.mb=256");
+        onHive().executeQuery("SET mapreduce.reduce.java.opts=-Xmx196m");
```

Comment:
`Xmx` is smaller than `memory.mb`. Is this not like  heap size vs our query-max-memory-per-node?

Reply:
I have read somewhere they suggest `Xmx` to be about 80% of `memory.mb`. JVM process can take more memory than `Xmx` so maybe `memory.mb` refers to whole maximum process memory consumption. I haven't checked that so far.